hack/windows/Prepare-Node.ps1

```diff
@@ -86,17 +102,23 @@ New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C
 $StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
 $global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")
 
-$netId = docker network ls -f name=host --format "{{ .ID }}"
+$global:containerRuntime = {{CONTAINER_RUNTIME}}
 
-if ($netId.Length -lt 1) {
-    docker network create -d nat host
-}
+if ($global:containerRuntime -eq "Docker") {
+    $netId = docker network ls -f name=host --format "{{ .ID }}"
+
+    if ($netId.Length -lt 1) {
+        docker network create -d nat host
+    }
 
-& C:\k\Prepare-ServiceInterface.ps1 -InterfaceAlias "HNS Internal NIC"
```

Comment:
We need this step for kube-proxy testbed.

Reply:
Reminder

Reply:
@NamanAg30 Will Prepare-ServiceInterface.ps1 be available for `StartKubeletFileContent` on kube-proxy testbed?

Reply:
change made